packages/cli/src/projects.test.ts

```diff
@@ -0,0 +1,252 @@
+import { MedplumClient } from '@medplum/core';
+import { MockClient } from '@medplum/mock';
+import { Project } from '@medplum/fhirtypes';
+import fs from 'fs';
+import { main } from '.';
+import { FileSystemStorage } from './storage';
+
+jest.mock('child_process');
+jest.mock('http');
+
+jest.mock('fs', () => ({
+  existsSync: jest.fn(),
+  readFileSync: jest.fn(),
+  writeFileSync: jest.fn(),
+  promises: {
+    readFile: jest.fn(async () => '{}'),
+  },
+}));
+
+let medplum: MedplumClient;
+
+describe('CLI Project', () => {
+  const env = process.env;
```

Comment:
I think you can delete all of the `process.env` saving and restoring, because you're not modifying env variables anywhere in the file.